teachopencadd/talktorials/T007_compound_activity_machine_learning/talktorial.ipynb

```diff
@@ -210,7 +210,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.            fpg = rdFingerprintGenerator.GetMorganGenerator(radius=2,fpSize=n_bits)
</pre><p>as</p><pre class="ql-syntax" spellcheck="false">Line #29.            fpg = rdFingerprintGenerator.GetMorganGenerator(radius=2, fpSize=n_bits)
</pre><p>(add space)</p><p> </p><p>-- same in the call below for radius 3</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/348/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>